spec/metric_fu/metrics/reek/generator_spec.rb

```diff
@@ -67,26 +47,56 @@
-ActivityReportsController#authorize_user calls current_user.primary_site_ids multiple times (Duplication)
-ActivityReportsController#authorize_user calls params[id] multiple times (Duplication)
-ActivityReportsController#authorize_user calls params[primary_site_id] multiple times (Duplication)
-ActivityReportsController#authorize_user has approx 6 statements (Long Method)
-
-"app/controllers/application.rb" -- 1 warnings:
-ApplicationController#start_background_task/block/block is nested (Nested Iterators)
-
-"app/controllers/link_targets_controller.rb" -- 1 warnings:
-LinkTargetsController#authorize_user calls current_user.role multiple times (Duplication)
-
-"app/controllers/newline_controller.rb" -- 1 warnings:
-NewlineController#some_method calls current_user.<< "new line\n" multiple times (Duplication)
-      HERE
+        @smells = [
+            double(source: "app/controllers/activity_reports_controller.rb",
+                   context: "ActivityReportsController#authorize_user",
+                   message: "calls current_user.primary_site_ids multiple times",
+                   smell_type: "Duplication",
+                   lines: [2, 4]),
+            double(source: "app/controllers/activity_reports_controller.rb",
+                   context: "ActivityReportsController#authorize_user",
+                   message: "calls params[id] multiple times",
+                   smell_type: "Duplication",
+                   lines: [5, 7]),
+            double(source: "app/controllers/activity_reports_controller.rb",
+                   context: "ActivityReportsController#authorize_user",
+                   message: "calls params[primary_site_id] multiple times",
+                   smell_type: "Duplication",
+                   lines: [11, 15]),
+            double(source: "app/controllers/activity_reports_controller.rb",
+                   context: "ActivityReportsController#authorize_user",
+                   message: "has approx 6 statements",
+                   smell_type: "Long Method",
+                   lines: [8]),
+
+            double(source: "app/controllers/application.rb",
+                   context: "ApplicationController#start_background_task/block/block",
+                   message: "is nested",
+                   smell_type: "Nested Iterators",
+                   lines: [23]),
+
+            double(source: "app/controllers/link_targets_controller.rb",
+                   context: "LinkTargetsController#authorize_user",
+                   message: "calls current_user.role multiple times",
+                   smell_type: "Duplication",
+                   lines: [8]),
+
+            double(source: "app/controllers/newline_controller.rb",
+                   context: "NewlineController#some_method",
+                   message: "calls current_user.<< \"new line\n\" multiple times",
+                   smell_type: "Duplication",
+                   lines: [6, 9])
+        ]
+        @lines = double(smells: @smells)
```

Comment:
any reason we need to make the smells double and fill with with smell doubles?  I'd rather use verified doubles if we're going to do that... not a blocker, but a followup PR